### PR TITLE
Add absolute change option in context column

### PR DESCRIPTION
**Pull request by bcolloran**
adds an "absolute change" option to the context menu.

requirement here: https://www.notion.so/rilldata/Expand-sorting-options-for-leaderboards-dimension-detail-tables-b4ec4ac635fd4634afff0c21f9779da4?pvs=4#116b22b05ac54bb5b02a5c3b61641290

NOTE: this should merge after  #2951

### CODE REVIEW

File: web-common/src/features/dashboards/dashboard-stores.ts [hunk 1]
```diff
@@ -492,14 +492,24 @@ const metricViewReducers = {
...
-      // NOTE: only show delta change if comparison is enabled
+      // NOTE: only show delta change if time comparison is enabled
       if (metricsExplorer.showComparison === false) return;
 
       metricsExplorer.leaderboardContextColumn =
         LeaderboardContextColumn.DELTA_PERCENT;
     });
   },
 
+  displayDeltaAbsolute(name: string) {
```
Comment (AdityaHegde) [inline web-common/src/features/dashboards/dashboard-stores.ts:503]: I am not sure if having a dedicated method for every option makes sense. Could we perhaps take the LeaderboardContextColumn value as argument?

File: web-common/src/features/dashboards/leaderboard/LeaderboardContextColumnMenu.svelte [hunk 2]
```diff
@@ -26,6 +26,8 @@
       metricsExplorerStore.displayDeltaChange(metricViewName);
     } else if (key === LeaderboardContextColumn.PERCENT) {
       metricsExplorerStore.displayPercentOfTotal(metricViewName);
+    } else if (key === LeaderboardContextColumn.DELTA_ABSOLUTE) {
```
Comment (AdityaHegde) [inline web-common/src/features/dashboards/leaderboard/LeaderboardContextColumnMenu.svelte:29]: This if-else chain can be moved over to switch-case. Easy to add more options. Applies to code in LeaderboardHeader and leaderboard-utils as well.

**Review by AdityaHegde — APPROVED** [review]
Looks good apart from a couple of small nits.

**Comment by bcolloran** [comment]
Thanks for the review @AdityaHegde , I made the changes you requested and will merge once CI passes

**Pull request merged**